addons/spreadsheet/static/src/list/plugins/list_ui_plugin.js

```diff
@@ -95,6 +95,13 @@ export class ListUIPlugin extends OdooUIPlugin {
                 this.lists[dataSourceId] = new ListDataSource(this.custom, listDefinition);
                 break;
             }
+            case "UPDATE_ODOO_LIST_SORTING": {
+                let listDefinition = { ...this.getters.getListModelDefinition(cmd.listId) };
+                listDefinition.searchParams.orderBy = cmd.orderBy;
```

Comment:
this line shouldn't be necessary since the definition should be already updated in the core plugin.
It can then be grouped with `UPDATE_ODOO_LIST_DOMAIN`


```js
case "UPDATE_ODOO_LIST_SORTING": 
case "UPDATE_ODOO_LIST_DOMAIN": {
```